examples/cis/main.tf

```diff
@@ -0,0 +1,79 @@
+provider "aws" {
+  region = var.region
+}
+
+module "test_label" {
+  source  = "cloudposse/label/null"
+  version = "0.22.0"
+
+  attributes = concat(module.this.attributes, ["test", "policy"])
+  context    = module.this.context
+}
+
+resource "aws_iam_role" "support_role" {
+  name = module.test_label.id
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"
+      },
+      "Action": "sts:AssumeRole",
+      "Condition": {}
+    }
+  ]
+}
+EOF
```

Comment:
Convert to data source

Reply:
5aae52b3d9554f64bd23f6448c59602dd4c780b2